packages/esm-service-queues-app/src/active-visits/active-visits-table.component.tsx

```diff
@@ -361,9 +361,6 @@ function ActiveVisitsTable() {
                           <TableCell className="cds--table-column-menu">
                             <EditMenu queueEntry={visitQueueEntries?.[index]} closeModal={() => true} />
                           </TableCell>
-                          <TableCell className="cds--table-column-menu">
-                            <OpenChartMenu patientUuid={visitQueueEntries?.[index]?.patientUuid} />
-                          </TableCell>
```

Comment:
I assume this is the "Profile" link you were mentioned removing?

Reply:
yep!